mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -389,7 +433,55 @@ func logModelGenerationSummary(modelToCompGenerateTracker *store.GenerticThreadS
 	utils.Log.Info(fmt.Sprintf("Generated %d models and %d components", totalAggregateModel, totalAggregateComponents))
 }
 
+func createJsonFile(resp *sheets.ValueRange) error {
+
+	var customResp []CustomValueRange
+
+	for _, row := range resp.Values[2:] {
+		if len(row) >= 15 && row[0] != "" {
```

Comment:
@RipulHandoo, this approach is fragile and breaks the flexibility of offered in the other publishing commands that work based on column name.

@MUzairS15, is that right?

Reply:
👀

Reply:
hmm. We'll need another set of eyeballs here.